script/main.js

```diff
@@ -0,0 +1,52 @@
+
+function getRandomIntInclusive(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+console.log(getRandomIntInclusive(1,6));
+
+const dice = {
+  value : 1,
+  roll: function(){
+    this.value = getRandomIntInclusive(1,6);
+    return this.value;
+  }
+}
+
+const diceTwo = {
+  value : 1,
+  roll: function(){
+    this.value = getRandomIntInclusive(1,6);
+    return this.value;
+  }
+}
+
+
+let button = document.querySelector('button').addEventListener('click', event => {
+  event.preventDefault();
+  let diceOneNumber = dice.roll();
+  // let containerOne = document.querySelector(".dice-01");
+
+
+  let newClass = "img-0"+diceOneNumber;
+  let elm = document.getElementById("dice-01");
+
+   if(elm.className !== newClass){
+       elm.className = newClass;
+}
+
+
+let diceTwoNumber = dice.roll();
+// let containerTwo = document.querySelector(".dice-02");
+
+
+let newClassTwo = "img-0"+diceTwoNumber;
+let elmTwo = document.getElementById("dice-02");
+
+ if(elmTwo.className !== newClassTwo){
+     elmTwo.className = newClassTwo;
+}
+});
+//just a comment to have one commit
```

Comment:
You need to actually make sure that you print the dice even without clicking the roll button. In order to do so, you will need to call the function that you set as the event handler for the click directly. The only way you can do that is by saving the function expression into a variable, and then referencing it in your event listener.

```javascript
// This defines the logic to roll and print the dice, same as what you currently have in your event handler function
const rollAndPrint = function(){
  // Event handler goes here...
}

// This is where we attache the named function expression into the event listener
document.querySelector('button').addEventListener('click', rollAndPrintDice);

// This is where we call the initial roll and print, so that the dice are also printed every time the page loads
rollAndPrintDice();
```